packages/core/src/getters/res-req-types.ts

```diff
@@ -87,35 +87,45 @@ export const getResReqTypes = (
           ] as ResReqTypesValue[];
         }
 
-        const formData = isFormData
+        const [formData, additionalFormDataImports] = isFormData
```

Comment:
If there are multiple return types, the responsibility for the function will no longer be single. And readability also decreases.

If possible, I would like to separate each function and separate the methods for obtaining `formData` and `additionalFormDataImports`.What do you think?

Reply:
Good idea. Please see new commit!